### PR TITLE
Adding skip test when the SMCP it's not the latest

### PR DESCRIPTION
**Pull request by fjglira**

### CODE REVIEW

File: pkg/util/env/env.go [hunk 1]
```diff
@@ -53,6 +53,16 @@ func GetSMCPVersion() version.Version {
 	return version.ParseVersion(getenv("SMCP_VERSION", "v2.5"))
 }
 
+func GetOperatorVersion() version.Version {
+	return version.ParseVersion(getenv("OPERATOR_VERSION", "v2.5"))
+}
```
Comment (luksa) [inline pkg/util/env/env.go:58]: Is this used anywhere? If not, it's better to remove it.

File: pkg/tests/ossm/smm_test.go [hunk 2]
```diff
@@ -89,6 +90,10 @@ func TestSMMReconciliation(t *testing.T) {
 		t.Log("This test verifies whether the member-of label is added back to the namespace")
 		t.Log("See https://issues.redhat.com/browse/OSSM-1397")
 
+		if !env.GetSMCPVersion().Equals(env.GetSupportedVersions()[2]) {
```
[the remark this answers is not in the document]
Comment (fjglira) [reply pkg/tests/ossm/smm_test.go:93]: Ok, I'll check a better way to approach this

**Comment by fjglira** [comment]
Hey @luksa I updated the PR by using now the GetOperatorVersion because this will tell us always the latest SMCP version

File: pkg/util/env/env.go (hunk 1 above)
Comment (fjglira) [reply pkg/util/env/env.go:58]: It's used now :)

File: pkg/util/version/version.go [hunk 3]
```diff
@@ -36,6 +36,14 @@ func ParseVersion(version string) Version {
...
 }
 
+func ParseVersions(versions string) []Version {
+	var result []Version
+	for _, version := range strings.Split(versions, " ") {
+		result = append(result, ParseVersion(version))
+	}
+	return result
+}
+
```
Comment (luksa) [inline pkg/util/version/version.go:46]: It looks like this function is not used anywhere, so it should be removed.

Comment (fjglira) [reply pkg/util/version/version.go]: Done

**Review by luksa — APPROVED** [review]

**Pull request merged**